client/stylesheets/_toCat.scss

```diff
@@ -3190,8 +3189,8 @@ On click the images will be displayed at normal size to complete the effect
 
 }
 
-#map-canvas {
-    height: 500px;
+.listingMapsCanvas {
+    height: 500px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces